src/runtime/graph/graph_runtime.cc

```diff
@@ -529,13 +184,32 @@ std::function<void()> GraphRuntime::CreateTVMOp(
       t->shape = &(arg_ptr->shape_data[i]);
     }
   }
+
   if (param.func_name == "__nop") {
     return [](){};
+  } else if (param.func_name == "__copy") {
+    // Perform cross device data copy.
+    // Directly copy data from the input to the output.
+    auto fexec = [arg_ptr]() {
+      DLTensor* from = static_cast<DLTensor*>(arg_ptr->arg_values[0].v_handle);
+      DLTensor* to = static_cast<DLTensor*>(arg_ptr->arg_values[1].v_handle);
+      TVM_CCALL(TVMArrayCopyFromTo(from, to, nullptr));
+    };
+    return fexec;
   }
+
   // get compiled function from module.
-  tvm::runtime::PackedFunc pf = module_.GetFunction(param.func_name, false);
+  tvm::runtime::PackedFunc pf =
+      runtime_host_module_.GetFunction(param.func_name, false);
+  if (pf == nullptr) {
+    for (const auto& it : runtime_device_mod_ctx_map_) {
+      pf = it.first->GetFunction(param.func_name, false);
```

Comment:
what if two mod have functions of same name?

Reply:
Thanks for pointing it out. This can be solved by using device information.